lib/screens/add_transaction_screen.dart

```diff
@@ -239,6 +239,7 @@ class TransactionsState extends State<AddTransactionScreen> {
           } else {
             List<int> tagIds = selectedTags!.map((tag) => tagList.indexOf(tag)).toList();
             await fileController.createTransaction(transactionName, transactionDate.toString(), tagIds, amount!, repeat);
+            await fileController.createTransaction(transactionName, transactionDate.toString(), tagIds, amount!, false);
```

Comment:
muss in einer if condition sein, sons werden 2 normalee transactions erstellt

Reply:
Mitdenken ist offensichtlich zu viel verlangt